app/models/user.server.ts

```diff
@@ -66,15 +66,15 @@ export async function createUser(email: User["email"], password: string) {
   const encrypted = Buffer.concat([cipher.update(lnmSecret), cipher.final()]);
 
   const encryptedLnmSecret = encrypted.toString("hex");
-  console.log("encrypted", encryptedLnmSecret);
 
   return prisma.user.create({
     data: {
       email,
       password: {
         create: {
           hash: hashedPassword,
-          lnmSecret: encryptedLnmSecret,
+          lnmSecret,
```

Comment:
i believe we should remove this `lnmSecret` from here, so that new users are only created w/ `encryptedLnmSecret`

Reply:
nevermind, good point about verifying its working first before wiping it